content/docs/use-cases/versioning-data-and-model-files/index.md

```diff
@@ -1,129 +1,103 @@
+  development.
+- Releasing: Tag stable data and models like you do with code. Employ semantic
+  versioning, connect branches to automatic deployment, etc.
+- Security: Git versioning of
+  [DVC metafiles](/doc/user-guide/dvc-files-and-directories) allows us to audit
+  data changes. And using cloud storage, data access control can be setup per
+  user or project.
+
+> ## ⚠️ DVC is not Git!
+>
+> DVC metafiles such as `dvc.yaml` and `.dvc` files serve as placeholders to
+> track data files and directories (among other purposes). They point to
+> specific data contents in the <abbr>cache</abbr>, providing the ability to
+> store multiple data versions out-of-the-box. Full-fledged
+> [version control](https://git-scm.com/book/en/v2/Getting-Started-About-Version-Control)
+> itself is left for Git to handle, however.
+
+## How it looks
+
+Working on data processing code, it's easy to overlook that the initial data
+also changes along the way. If something goes wrong (or for any other reason),
+it's easy to revisit previous versions of the code with Git. But the results may
+still be different if using the latest data.
+
+To "eliminate this variable" and achieve full
+[reproducibility](/doc/start/data-pipelines), we can use `dvc add` to capture
+data snapshots that match code checkpoints:
 
 ```dvc
-$ dvc repro
-Running stage 'train' with command:
-        python train.py images/
-Updating lock file 'dvc.lock'
+$ git add cleanup.c
+$ dvc add data/raw
 ...
+$ git add data/raw.dvc data/.gitignore
 
-$ git add dvc.yaml dvc.lock .gitignore
-$ git commit -m "Train model via DVC."
-$ git tag -a "v1.0" -m "Fist model"   # We'll use this soon ;)
+$ git commit -m 'Data cleanup v1.0'
 ```
 
-> See also `dvc.lock`.
-
-## Switching versions
-
-After iterating on this process and producing several versions, you can combine
-`git checkout` and `dvc checkout` to perform full or partial
-<abbr>workspace</abbr> restorations.
-
-![](/img/versioning.png) _Code and data checkout_
-
-> Note that `dvc install` enables auto-checkouts of data after `git checkout`.
-
-A full checkout brings the whole <abbr>project</abbr> back to a previous version
-— code, dataset and model files all match each other:
+Now that Git is tracking the code (including a `.dvc` file created by DVC), and
+DVC is tracking the data, we can repeat the procedure to generate more commits.
+Use `dvc checkout` to go back:
 
 ```dvc
 $ git checkout v1.0
 $ dvc checkout
-M       images
-M       model.pkl
+M       data\raw
 ```
 
-However, we can checkout certain parts only, for example if we want to keep the
-latest source code and model but rewind to the previous dataset only:
+![](/img/versioning.png) _Full project restoration_
 
-```dvc
-$ git checkout v1.0 images.dvc
-$ dvc checkout images.dvc
-M       images
-```
+> For more hands-on experience, we recommend following the
+> [versioning tutorial](/doc/use-cases/versioning-data-and-model-files).
+
+## Versioned storage
```

Comment:
this whole section looks disconnected at this moment

Reply:
Yeah I know. It's linked from the top (paragraph about storage) but it should be better in the next iteration... ⏳

Reply:
I think the section flow seems more natural now that DVC is not Git! is gone. PTAL